bin/express-cli.js

```diff
@@ -51,8 +51,9 @@ program
   .option('-e, --ejs', 'add ejs engine support', renamedOption('--ejs', '--view=ejs'))
   .option('    --pug', 'add pug engine support', renamedOption('--pug', '--view=pug'))
   .option('    --hbs', 'add handlebars engine support', renamedOption('--hbs', '--view=hbs'))
+  .option('    --jsx', 'add react jsx engine support', renamedOption('--jsx', '--view=jsx'))
```

Comment:
You don't want to ad a new option that is immediately deprecated :) You can remove this `--jsx` option 👍

Reply:
the --engine are for deprecated engines ?

Reply:
I'm not sure what exactly you mean. When you run `express --jsx` in your PR you have it printing a warning saying that `--jsx` should no longer be used and use `--view=jsx` instead. Just don't add it at all, there doesn't seem to be a point.

Reply:
kk